client/src/main/java/com/arcbees/beeshop/client/application/ApplicationView.java

```diff
@@ -16,17 +16,25 @@
 
 package com.arcbees.beeshop.client.application;
 
+import static com.google.gwt.query.client.GQuery.$;
```

Comment:
I think we should open a PR with a "reformat all" or something like that to avoid further conflicts/reformats